src/openforms/js/components/form/content.js

```diff
@@ -26,19 +26,80 @@ const CONTENT_PRESENTATION = {
   components: [SHOW_IN_EMAIL, SHOW_IN_PDF],
 };
 
+const CONTENT_EDITOR = {
+  weight: 0,
+  type: 'textarea',
+  editor: 'ckeditor',
+  label: 'Content',
+  hideLabel: true,
+  input: true,
+  key: 'html',
+  as: 'html',
+  rows: 3,
+  tooltip: 'The HTML template for the result data items.',
+};
+
 const CONTENT_EDIT_TABS = {
   components: [
     {
-      weight: 0,
-      type: 'textarea',
-      editor: 'ckeditor',
-      label: 'Content',
-      hideLabel: true,
-      input: true,
-      key: 'html',
-      as: 'html',
-      rows: 3,
-      tooltip: 'The HTML template for the result data items.',
+      type: 'columns',
+      key: 'content',
+      columns: [
+        {
+          components: [
+            {
+              type: 'tabs',
+              components: [{label: 'Content', components: [CONTENT_EDITOR]}],
+            },
+          ],
+        },
+        {
+          components: [
+            {
+              ...TRANSLATIONS,
+              hideLabel: true,
+              components: [
+                {
+                  key: 'languages',
+                  type: 'tabs',
+                  components: LANGUAGES.map(([languageCode, _label]) => {
+                    return {
+                      key: languageCode,
+                      label: languageCode.toUpperCase(),
+                      components: [
+                        {
+                          label: 'Literal',
+                          hideLabel: true,
+                          key: `of-translations.${languageCode}[0].literal`,
```

Comment:
This magnificent key makes it such that it looks like the datagrid used else where and gets picked up into the componentTranslations on PUT and POST.